README.rst

```diff
@@ -9,7 +9,7 @@ without leaving the command line.
 Requirements
 ------------
 
-* Python 2.7+ or Python 3.3+
+* Python 2.7+ or Python 3.3+ <= 3.6
```

Comment:
I'm curious why we need to restrain Python version here, is there a reason?

Reply:
tried pytest with python 3.7 and tests were failing 
all tests were failing to initialize 

plus travis, appvoyer and tox file, all runs on version less than 3.7

may be one for separate PR
error log
```
=================================================================== ERRORS ===================================================================
___________________________________________________ ERROR collecting tests/test_deploy.py ____________________________________________________
../../venv/shub3/lib/python3.7/site-packages/_pytest/python.py:177: in pytest_pycollect_makeitem
    raise StopIteration
E   StopIteration

The above exception was the direct cause of the following exception:
../../venv/shub3/lib/python3.7/site-packages/_pytest/runner.py:163: in __init__
    self.result = func()
../../venv/shub3/lib/python3.7/site-packages/_pytest/main.py:476: in _memocollect
    return self._memoizedcall('_collected', lambda: list(self.collect()))
../../venv/shub3/lib/python3.7/site-packages/_pytest/main.py:347: in _memoizedcall
    res = function()
../../venv/shub3/lib/python3.7/site-packages/_pytest/main.py:476: in <lambda>
    return self._memoizedcall('_collected', lambda: list(self.collect()))
../../venv/shub3/lib/python3.7/site-packages/_pytest/python.py:412: in collect
    return super(Module, self).collect()
../../venv/shub3/lib/python3.7/site-packages/_pytest/python.py:326: in collect
    res = self.makeitem(name, obj)
../../venv/shub3/lib/python3.7/site-packages/_pytest/python.py:338: in makeitem
    collector=self, name=name, obj=obj)
../../venv/shub3/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py:745: in __call__
    return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
../../venv/shub3/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py:339: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
../../venv/shub3/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py:334: in <lambda>
    _MultiCall(methods, kwargs, hook.spec_opts).execute()
../../venv/shub3/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py:613: in execute
    return _wrapped_call(hook_impl.function(*args), self.execute)
../../venv/shub3/lib/python3.7/site-packages/_pytest/vendored_packages/pluggy.py:250: in _wrapped_call
    wrap_controller.send(call_outcome)
E   RuntimeError: generator raised StopIteration
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================== 1 error in 0.68 seconds ===========================================================
```

Reply:
pytest is outdated as well

Reply:
many dependencies are outdated and either throwing error or warning
pytest
pipenv
click
scrapinghub ( throwing abc meta warning)

Reply:
I see, thanks for checking. You're right, many dependencies are outdated, but that should be addressed separately, let's not mix the different things together and focus on the sources thing here